internal/pkg/deploy/cloudformation/stack/stack_local_integration_test.go

```diff
@@ -26,7 +25,12 @@ const (
 )
 
 func Test_Autoscaling_Integration(t *testing.T) {
```

Comment:
I wonder if we need a separate `Test_TaskDefinitionOverride_Integration` instead of adding it to this autoscaling integ test, just for clarity?

Reply:
Oooh sorry for the confusion. I should rename this function because it is not only for auto scaling validation now.